javascript/organizing_your_javascript_code/es6_modules.md

```diff
@@ -64,9 +64,7 @@ There are a couple of key concepts to understanding how webpack works - __entry_
 
 Now that we (sorta) understand what webpack is doing it's time to discuss the module syntax. There are only 2 components to it - __import__ and __export__.
 
-- Take a moment to look at the docs for [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [export](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export).
-
-Of course, the import statement is the same thing that you used during the webpack tutorial! These things are simple to use.
+The import statement is the same thing that you used during the webpack tutorial! These things are simple to use.
```

Comment:
```suggestion
The import statement is the same thing that you used during the webpack tutorial so it should be familiar by now.
```
nit: in case someone doesn't find this simple :)